src/serac/physics/utilities/functional/detail/segment_H1.inl

```diff
@@ -24,6 +24,9 @@ struct finite_element<Geometry::Segment, H1<p, c> > {
   static constexpr int  dim        = 1;
   static constexpr int  ndof       = (p + 1);
 
+  using residual_type =

```

Comment:
`residual_type` is part of the `finite_element` concept. It tells the finite element kernel templates what the type of the element residual vector should be (dictated by the elements from the test space). Using an abridged version of one of the kernels in boundary_integral.hpp as an example


```cpp
  using element_residual_type = typename trial_element::residual_type;

  // for each element in the domain
  for (int e = 0; e < num_elements; e++) {
   
    // this is where we will accumulate the element residual tensor
    element_residual_type r_elem{};

    // for each quadrature point in the element
    for (int q = 0; q < static_cast<int>(rule.size()); q++) {
      ...
      r_elem += residual_contribution_from_this_quadrature_point;
      ...
    }
    detail::Add(r, r_elem, e);
  }
```

It only got added in this PR, because the segment elements were previously unused, so the error went unnoticed.